src/views/v2/base.php

```diff
@@ -1,7 +1,28 @@
+<?php
+/**
+ * View: Base View
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/base.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version 4.9.4
```

Comment:
Let's set the version TBD

Reply:
This is there from version 1, I just forgot to set it before.

Reply:
Yeah, I know. But just mentioning because the point of the `@version` is to highlight if there have been changes every time the plugin updates.